x-pack/plugins/observability/public/pages/slos/components/compact_view/slo_list_compact_view.tsx

```diff
@@ -402,3 +406,11 @@ export function SloListCompactView({ sloList, loading, error }: Props) {
     </>
   );
 }
+
+const LOADING_SLOS_LABEL = i18n.translate('xpack.observability.slo.loadingSlosLabel', {
+  defaultMessage: 'Loading SLOs ...',
+});
+
+const NOT_AVAILABLE_LABEL = i18n.translate('xpack.observability.slo.notAvailableLabel', {
+  defaultMessage: 'No SLOs found',
+});
```

Comment:
The copy "No SLOs found" is not very appropriate for where it is used:
![image](https://github.com/elastic/kibana/assets/1376800/5c964075-5b95-4506-8f09-0f73d050d709)

I would use one of these depending on the column, but maybe just "N/A" is enough for all.
- for the instance column: "No instance", "None" or "N/A" or just "-"
- For the SLI column: "No data" or "N/A"
- For the error budget column: "No data" or "N/A"

Reply:
@kdelemme that's a mistake i only meant to show table no data state. i guess i ended up reusng some label. let me correct it